src/main/java/net/sf/rails/game/TrackConfig.java

```diff
@@ -132,6 +132,7 @@ public String toString() {
      * a description of the connections
      * @deprecated Use {@link #getConnectionString(MapHex,Tile,HexSide,Station)} instead
      */
+    @Deprecated
```

Comment:
LOL, I was wondering how long it would take for someone to fix this. Evidently not long!